sw/device/lib/dif/dif_rv_core_ibex.h

```diff
@@ -201,7 +201,7 @@ typedef struct dif_rv_core_ibex_crash_dump_info {
   /**
    * The crash dump state for the current execution.
    */
-  dif_rv_core_ibex_crash_dump_state_t fault_stage;
+  dif_rv_core_ibex_crash_dump_state_t fault_state;
```

Comment:
Nit. I would move this commit to the second position in the PR.

Reply:
Yes that would be neater and...     done.